steps/steps_requirements.txt

```diff
@@ -1,10 +1,6 @@
-qiskit==0.18.3
-qiskit-ibmq-provider==0.6.1
-pyquil==2.17.0
-numpy>=1.18.1
-git+https://github.com/zapatacomputing/qe-qiskit#dev
-git+https://github.com/zapatacomputing/qe-qulacs#dev
-git+https://github.com/zapatacomputing/qe-forest#dev
-git+https://github.com/zapatacomputing/z-quantum-core.git#dev
-git+https://github.com/zapatacomputing/qe-openfermion#dev
+git+https://github.com/zapatacomputing/qe-qiskit@dev
+git+https://github.com/zapatacomputing/qe-qulacs@dev
+git+https://github.com/zapatacomputing/qe-forest@dev
+git+https://github.com/zapatacomputing/z-quantum-core.git@dev
+git+https://github.com/zapatacomputing/qe-openfermion@dev
```

Comment:
Is this because these packages defined their own versions of Qiskit etc?

Reply:
Yes, rather than pinning here packages explicitly, we would now depend on how they are pinned in the packages. This is required so that if there's a dependency change there, then we wouldn't have to change the requirements in this file.

The other change was adjusting the way how the `dev` branch is used (previously had the incorrect syntax).